### PR TITLE
Ephys example scripts

### PR DESCRIPTION
**Pull request by jsiegle**
Add examples scripts and generated JSON files for `ephys_session` and `ephys_rig`

### CODE REVIEW

File: examples/ephys_session.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+session = EphysSession(experimenter_full_name="Josh Siegle",
+                      subject_id=100001,
+                      session_start_time = datetime.now(),
+                      session_end_time=datetime.now(),
+                      session_type="Test",
+                      rig_id="323_EPHYS1",
+                      data_streams=[stream])
+
+with open("ephys_session.json", "w") as outfile:
```
Comment (dyf) [inline examples/ephys_session.py:41]: Can you replace this with `session.write_standard_file()`?

File: examples/ephys_session.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from aind_data_schema.ephys.ephys_session import *
```
Comment (dyf) [inline examples/ephys_session.py:2]: Can you `import ... as es` instead to be consistent with the other examples?

File: examples/ephys_rig.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+rig = EphysRig(rig_id="323_EPHYS1",
+              ephys_modules=[ephys_moduleA, ephys_moduleB],
+              cameras=[camassm1, camassm2],
+              laser_modules=[laser_module],
+              daqs=[basestation, harp],
+              stick_microscopes=[microscope],
+              mouse_platform=running_wheel)
+
+with open("ephys_rig.json", "w") as outfile:
```
Comment (dyf) [inline examples/ephys_rig.py:136]: Can you replace this with `rig.write_standard_file()`?

**Review by dyf — CHANGES_REQUESTED** [review]

File: examples/ephys_session.py (hunk 2 above)
Comment (dyf) [reply examples/ephys_session.py:2]: (the `import *` is why the CI is breaking right now)

**Comment by dyf** [comment]
@jsiegle almost there - now you need to regenerate your .json files.

**Comment by jsiegle** [comment]
Hmm...I regenerated but it's still failing. And the expected vs. actual JSON strings look the same. Can you tell what the difference is?

**Comment by dyf** [comment]
@jsiegle It's a couple things.  
1) the datetimes don't match because you are using `datetime.now()`.  Can you fix the time like this? https://github.com/AllenNeuralDynamics/aind-data-schema/blob/83de3c9b189e7a5fd425af385bf79b3287ddb74b/examples/subject.py#L6
2) subject ID should be a string, not an integer.  I'm not sure why that's rendering out as an integer locally and a string in the CI.

**Comment by jsiegle** [comment]
Thanks, it's passing now!

**Review by dyf — APPROVED** [review]

**Pull request merged**